jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -1,104 +1,121 @@
 
-The following sections will describe the CircleCI specific semantics, what is a CircleCI Decision, and how to use the CircleCI-CLI to manage your policies.
+Config policy decisions are stored and can be audited. This provides useful data about the pipeline defintions being run within your organization.
 
-#TODO: A section on the architecture/process - how a config is evaluated 
+== How config policy management works
 
-## Writing Rego Policies using CircleCI Domain-Specific Language
-{: #writing-rego-policies }
+Config policy management uses a decision engine based on link:https://www.openpolicyagent.org/[Open Policy Agent (OPA)]. Policies are written in the `rego` query language, as defined by OPA. Rule violations are surfaced when pipelines are triggered.
 
-# TODO: re-read to ensure no confusion between rego rules and CircleCI violations
+#TODO: A section on the architecture/process - how a config is evaluated#
 
-In order for CircleCI to make decisions about configs, it needs to be able to interpret the output 
-generated by policy evaluation. To do this, a policy must be written to meet CircleCI specifications. 
-Policies must be written in `rego`, a purpose-built declarative policy language that supports Open Policy 
-Agent (OPA). You can find more information about `rego` [here](https://www.openpolicyagent.org/docs/latest/policy-language/).
+== Writing rego policies using CircleCI domain-specific language
+
+#TODO: re-read to ensure no confusion between rego rules and CircleCI violations#
+
+Policies are written in `rego`, a purpose-built declarative policy language that supports OPA. You can find more about `rego` in the link:https://www.openpolicyagent.org/docs/latest/policy-language/[`rego` language docs].
+
+In order for CircleCI to make decisions about your configs, it needs to be able to interpret the output 
+generated when your policies are evaluated, therefore, Policies must be written to meet the CircleCI specifications detailed on this page. 
+
+=== Package and name
 
 All policies must belong to the `org` package and declare the policy name as the first rule. All policy rego files should include:
 
-```rego
+[source,rego]
+----
 package org
 
 policy_name["unique_policy_name"]
-```
+----
 
-The `policy_name` is an alphanumeric string with max length of 80 characters. It is used to uniquely identify the policy by name within the system, similar to a kubernetes named resource.
-Policy names are required to be unique and as such it is not allowed to create two policies with the same name.
+The `policy_name` is an alphanumeric string with a maximum length of 80 characters. It is used to uniquely identify the policy by name within the system (similar to a Kubernetes named resource).
+Policy names must be unique. Two policies cannot have the same name within an organization.
 
 The `policy_name` must be declared using a partial rule and declare the name as a `rego key`: `policy_name["NAME"]`.
```

Comment:
@sagar-connect what is meant by "partial rule" here?

Reply:
In this context, we are trying to demonstrate the syntax for defining the policy name rule, which is:

`policy_name["name_string"]`

More details about partial rule:
Partial rules are rules that evaluate to partial results on evaluation i.e. the result is an aggregation of all the results for that key. https://medium.com/@agarwalshubhi17/rego-cheat-sheet-5e25faa6eee8#:~:text=evaluate%20to%20true.-,Partial%20rules,-Generating%20sets%3A%20Head

Reply:
And, as you rightly pointed out, this is also true for the policy_name rule:

> Oh I guess it's that it doesn't following the three stage process: evaluation, enforcement, enablement